bigquery/indexer.py

```diff
@@ -462,6 +476,8 @@ def main():
     bq_client = bigquery.Client(project=deploy_project_id)
     storage_client = storage.Client(project=deploy_project_id)
 
+    all_table_documents = {}
```

Comment:
Instead of separate `all_table_documents` and `all_sample_documents` dicts, just put the sample dicts directly inside participant dicts, like how they will be stored into Elasticsearch.

Instead of `all_table_documents` and `all_sample_documents`, have a single `es_docs` variable:
```
    # A dict containing documents to be indexed into Elasticsearch.
    # Key is participant id; value is document contents.
    es_docs = {}
    for table_name in bigquery_config['table_names']:
        table = read_table(bq_client, table_name)
        index_fields(es, fields_index_name, table, sample_id_column)
        create_mappings(es, index_name, table_name, table.schema,
                        participant_id_column, sample_id_column,
                        sample_file_columns)
        add_table_to_es_docs(es, bq_client, storage_client, index_name, table,
                    participant_id_column, sample_id_column,
                    sample_file_columns, deploy_project_id, es_docs)
    indexer_util.bulk_index(es, index_name, es_docs)
```